components/workExperience/CompanyCard.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import { useState } from "react";
+import { motion } from "framer-motion";
+
+import { companies } from "../icons";
+import { jobs } from "@/constants";
+
+const CompanyCard = () => {
+  const [hoveredCard, setHoveredCard] = useState<number | null>(null);
+
+  const handleCardHovered = (index: number) => {
+    setHoveredCard(index);
+  };
+
+  const handleCardLeave = () => {
+    setHoveredCard(null);
+  };
+
+  return (
+    <>
+      {companies.map((CompanyLogo, index) => (
+        <motion.div
```

Comment:
The same thing here. map over and return a reusable component here. Also, there is no reason to use a key as you currently have it. If this is necessary for NextUi, please forgive my suggestion...

Reply:
I've made a reusable component and map over that. Thanks!